Gutenberg/cocoapods_helpers.rb

```diff
@@ -155,3 +160,14 @@ def react_native_path!(gutenberg_path:)
 
   react_native_path
 end
+
+def react_native_version!(gutenberg_path:)
+  react_native_path = react_native_path!(gutenberg_path: gutenberg_path)
+  package_json_path = File.join(react_native_path, 'package.json')
+  package_json_content = File.read(package_json_path)
+  package_version_match = package_json_content.match(/"version":\s*"(.*?)"/)
```

Comment:
Nitpick: Ruby has built in support for parsing JSONs

```suggestion
  package_json_version = JSON.parse(package_json_content)['version']
```

This will also required importing Ruby's JSON module

```ruby
# At the top of the file
require 'json'
```

Reply:
Yep, I was tempted to use the JSON library but went that way (for no particular reason) to avoid adding it to [`Gemfile`](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/Gemfile).

Reply:
Even if you indeed need to explicitly `require 'json'` to use it, iinm the JSON module is part of Ruby core, so there's no need to add it to the `Gemfile`.

_(Think of if you were using `import Foundation` in a Swift project 😛 )_

Reply:
Ah, thanks for pointing that out @AliSoftware. Not sure why I thought it wasn't part of the core. I'll use this approach then.

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/21021/commits/80db7bae5854dd473294a059d6673be79a0d8f56.